controllers/helpers/helpers.go

```diff
@@ -268,3 +268,13 @@ func IsImmutableError(err error) bool {
 
 	return false
 }
+
+func setLabels(from, to controllerClient.Object) {
```

Comment:
```suggestion
func copyLabels(from, to controllerClient.Object) {
```

Makes more sense and easier to understand

Reply:
Done